### DISCUSSION TITLE
A monthly service option for the Smart Connect GPTs

### DISCUSSION THREAD

**Discussion by brianpetro** [brianpetro/obsidian-smart-connections#526] [Announcements]
Would you like to access the Smart Connect GPTs?

Would you prefer there to be a monthly service option?

Please let me know if you are interested in this.

Thanks for your help
🌴

**Comment by conreid-838** [comment]
Interested!

**Comment by Allwaysthismoment** [comment]
Good evening and I would definitely be interested.

To be honest though, I'm so greatful for this plugin, if I knew for sure I could use my local (Ollama or LM) with v2.1, I'd pay the supporter fee now just to get going.

**Reply by brianpetro** [reply]
Hey @Allwaysthismoment 

Happy to see that you like Smart Connections 😊 

This comment has a couple notes on where v2.1 is at wrt local chat models https://github.com/brianpetro/obsidian-smart-connections/issues/302#issuecomment-2028092945

🌴

**Comment by Flintr** [comment]
I really want to test out Smart Connect GPT, but the up front $100 minimum is a big hurdle without knowing experiencing it first. A monthly service option would be a great way to try before you buy.

**Reply by christianngarcia** [reply]
Monthly option is coming soon, but you can try it for free, see this video https://youtu.be/TDUWvOc7xJo?si=Buai2Vb7cMAPDDxF

**Comment by andrewf13** [comment]
Any updates on the subscription model please?
I think a few dollars/month would be very nice price point.

**Reply by christianngarcia** [reply]
Monthly option is coming soon, but you can try it for free, see this video https://youtu.be/TDUWvOc7xJo?si=Buai2Vb7cMAPDDxF
